scanomatic/data_processing/phenotyper.py

```diff
@@ -537,8 +544,16 @@ def LoadFromNumPy(
                 times_data_path += ".npy"
 
         return cls(
-            unpickle_with_unpickler(np.load, data_directory),
-            unpickle_with_unpickler(np.load, times_data_path),
+            unpickle_with_unpickler(
+                np.load,
+                data_directory,
+                allow_pickle=True,
+            ),
+            unpickle_with_unpickler(
+                np.load,
+                times_data_path,
+                allow_pickle=True,
```

Comment:
Are all uses of this function now calling it with allow_pickle=True? If so consider removing the argument and simplifying that function.

Reply:
`unpickle_with_unpickler` is not, or is atleast not meant to, be called with `np.load`, to which `allow_pickle` is an argument. I'll have a look. If it is only used with `np.load` I guess I can remove that argument too.

Reply:
As far as I can tell, it is used exclusively with `np.load`. One way forward is to replace each call to `unpickle_with_unpickler` with 
```python
np.load(
    safe_load(p),
    allow_pickle=True,
)
```
and remove `unpickle_with_unpickler` entirely. Either that, or renaming it `safe_load_npy` and doing precisely that, but save on arguments in the bulk of the code. What do you think?

Reply:
if that  is all that it does now, then I'm all for removing the unpickle method and just use np.load in code

Reply:
fixed